src/test/java/HomeWork/pages/FileUploadPage.java

```diff
@@ -2,13 +2,14 @@
 
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
 
 public class FileUploadPage extends BasePage {
-    public final static By UPLOAD_BUTTON = By.cssSelector("input[id=file-submit]");
-    public final static By FILE_UPLOADED_SUCCESS = By.cssSelector("div[class='panel text-center']");
-    public final static By FILE_UPLOADED_PAGE = By.xpath("//*[@id=\"content\"]/div/h3");
-    public static String PATH_TO_LOADING_FILE = System.getProperty("user.dir") + "/src/test/resources/Test";
-
+    private final static By UPLOAD_BUTTON = By.cssSelector("input[id=file-submit]");
+    private final static By FILE_UPLOADED_SUCCESS = By.cssSelector("div[class='panel text-center']");
+    private final static By FILE_UPLOAD_FIELD = By.cssSelector("input[id=file-upload]");
+    private final static By FILE_UPLOADED_PAGE = By.xpath("//*[@id=\"content\"]/div/h3");
+    private static final String PATH_TO_LOADING_FILE = System.getProperty("user.dir") + "/src/test/resources/Test";
```

Comment:
Again, the path to file is a test data - it shouldn't be hardcoded inside the page class, it should be passed as an input argument to enterFileUploadField() method